web/models/users.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Notox.User do
+  use Notox.Web, :model
+
+  @required_fields [
+    :username
```

Comment:
I'll add the comma. When I look at this again I'm wondering why I don't need to add `:password` in here. I tried leaving the password field blank and I get a flash saying it can't be blank, but I don't know why.